treeherder/config/settings.py

```diff
@@ -505,53 +505,42 @@ def server_supports_tls(url):
 PULSE_DATA_INGESTION_SOURCES = env.json(
     "PULSE_DATA_INGESTION_SOURCES",
     default=[
-        # {
-        #     "name": "exchange/taskcluster-treeherder/v1/jobs",
-        #     "projects": [
-        #         'mozilla-central',
-        #         'mozilla-inbound'
-        #         # other repos TC can submit to
-        #     ],
-        #     "destinations": [
-        #         'production'
-        #         'staging'
-        #     ]
-        # },
-        # {
-        #     "name": "exchange/treeherder-test/jobs",
-        #     "projects": [
-        #         'mozilla-inbound'
-        #     ],
-        #     "destinations": [
-        #         'production'
-        #         'staging'
-        #     ]
-        #
-        # }
+        {
+            "exchange": "exchange/taskcluster-treeherder/v1/jobs",
+            "projects": [
+                'mozilla-central.#',
+                'mozilla-inbound.#'
+                # other repos TC can submit to
+            ],
+            "destinations": [
+                'production'
+                # 'staging',
+            ]
+        }
         # ... other CI systems
     ])
 
 PULSE_PUSH_SOURCES = env.json(
     "PULSE_PUSH_SOURCES",
     default=[
-        # {
-        #     "exchange": "exchange/taskcluster-github/v1/push",
-        #     "routing_keys": [
-        #         '#'
-        #     ],
-        # },
-        # {
-        #     "exchange": "exchange/taskcluster-github/v1/pull-request",
-        #     "routing_keys": [
-        #         '#'
-        #     ],
-        # },
-        # {
-        #     "exchange": "exchange/hgpushes/v1",
-        #     "routing_keys": [
-        #         "#"
-        #     ]
-        # },
+        {
+            "exchange": "exchange/taskcluster-github/v1/push",
+            "routing_keys": [
+                '#'
+            ],
+        },
+        {
+            "exchange": "exchange/taskcluster-github/v1/pull-request",
+            "routing_keys": [
+                '#'
+            ],
+        },
+        {
+            "exchange": "exchange/hgpushes/v1",
+            "routing_keys": [
+                "#"
+            ]
+        }
```

Comment:
I uncommented all the default exchanges, if that makes sense.

Reply:
Yep, this looks right to me.